cleanlab_studio/utils/databricks_utils.py

```diff
@@ -104,12 +105,10 @@ def create_df_based_imageset_archive(df, archive_name=None) -> str:
         first_row = True
         for row in df.toLocalIterator():
             row = row.asDict()
-            original_path = dbfs_to_posix_path(row[image_col].origin)
+            original_path = dbfs_to_posix_path(Path(row[image_col].origin))
             del row[image_col]
             path_in_zip = (
-                Path(archive_name)
-                .joinpath(Path(original_path).resolve(strict=False).name)
-                .as_posix()
+                Path(archive_name).joinpath(original_path.resolve(strict=False).name).as_posix()
```

Comment:
nit: can we just initialize `archive_name` as a `pathlib.Path()` instead of converting it all these separate times?

Reply:
`archive_name` should just be the name of the zip. Makes more sense to keep as `str`.